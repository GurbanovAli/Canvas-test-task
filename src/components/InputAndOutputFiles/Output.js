import React from 'react';
import { useAppState } from '../ControlFile';


function Output() {
   const state = useAppState();
   const {
      loading = false, getFile = 'No file'
   } = state;
   return (
         <div
           className = 'output' >
           { loading }
           <pre> { getFile } </pre> 
         </div>
   );
}

export default Output;
