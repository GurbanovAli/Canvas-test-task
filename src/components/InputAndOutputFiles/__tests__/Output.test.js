import React from 'react';
import { render } from '@testing-library/react';
import Output from '../Output';
import Control from '../../ControlFile';


test('<Output /> spec', () => {
    const { container } = render(
      <Control>
        <Output />
      </Control>
    );
    expect(container).toMatchSnapshot();
  });
