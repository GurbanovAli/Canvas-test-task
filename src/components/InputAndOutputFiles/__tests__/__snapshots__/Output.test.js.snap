// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Output /> spec 1`] = `
<div>
   
  <div
    class="output"
  >
    <pre>
       
      No file
       
    </pre>
  </div>
   
</div>
`;
